service-runner/src/main/kotlin/org/stellar/anchor/platform/run_profiles/RunKotlinReferenceServer.kt

```diff
@@ -4,9 +4,20 @@ package org.stellar.anchor.platform.run_profiles
 
 import kotlinx.coroutines.runBlocking
 import org.stellar.anchor.platform.*
+import org.stellar.reference.di.ConfigContainer.Companion.KT_REFERENCE_SERVER_CONFIG
 
 fun main() = runBlocking {
-  testProfileExecutor = TestProfileExecutor(TestConfig())
+  testProfileExecutor =
```

Comment:
Why is this also required in `RunTestProfile.kt`?

Reply:
This is to default KT_REFERENCE_SERVER_CONFIG to "service-runner/src/main/resources/config/reference-config.yaml" to reduce the verboseness in running the test profiles.